src/index.js

```diff
@@ -1 +1,4 @@
 export { createHeadlessForm } from './createHeadlessForm';
+
+export { pickXKey } from './internals/helpers';
```

Comment:
@brennj I cherry-picked the commit we made yesterday at #3 to this PR, so that #3 can remain focused on just release scripts. WDYT?